cpp/src/aztec3/circuits/rollup/root/.test.cpp

```diff
@@ -270,50 +291,77 @@ TEST_F(root_rollup_tests, blabber)
     }
 
     // Compute a sibling path for the new commitment subtree. Get the full first, and then shorten it.
```

Comment:
you could instead do something similar to base:
```
template <size_t N>
std::array<fr, N> get_sibling_path(stdlib::merkle_tree::MemoryTree tree, size_t leafIndex, size_t subtree_depth_to_skip)
{
    std::array<fr, N> siblingPath;
    auto path = tree.get_hash_path(leafIndex);
    // slice out the skip
    leafIndex = leafIndex >> (subtree_depth_to_skip);

    for (size_t i = 0; i < N; i++) {
        if (leafIndex & (1 << i)) {
            siblingPath[i] = path[subtree_depth_to_skip + i].first;
        } else {
            siblingPath[i] = path[subtree_depth_to_skip + i].second;
        }
    }
    return siblingPath;
}
```

`get_sibling_path< PRIVATE_DATA_SUBTREE_INCLUSION_CHECK_DEPTH >(data_tree, 0, PRIVATE_DATA_SUBTREE);`
Do same for `contracts`. 
For historic trees just set the final argument to 0